v2/pkgs/tools.go

```diff
@@ -47,36 +49,37 @@ import (
 //	        └── 1.7.0
 //	            └── bossac
 //
-// It requires an Indexes client to list and read package index files: use the Indexes struct
+// It requires an Index Resource to search for tools
 type Tools struct {
-	Indexes interface {
-		List(context.Context) ([]string, error)
-		Get(context.Context, string) (Index, error)
-	}
+	Index  *index.Resource
 	Folder string
 }
```

Comment:
```suggestion
type Tools struct {
	index  *index.Resource
	folder string
}

func New(index *index.Resource, folder string) *Tools {
   return &Tools{
       index: index,
       folder: folder,
   }
}
```
nit: would be great to make such properties unexported and use a "constructor" function